news/5815.feature.rst

```diff
@@ -0,0 +1,2 @@
+(Windows) Add support for discovering the Python shared library in the
```

Comment:
Can we say something here to make it clear that using a `venv` is still required. Currently, it kind of implies that this shared library is the only problem with the windows store Python.

Reply:
Hmm, actually, the issues in non-venv are caused by the same empty shims (in a different part of code, which uses only `sys.executable` instead of trying to use `sys._base_executable` - and since venv provides non-empty copy of python.exe, that part manages to get through when venv is used).

I guess this is a good opportunity to sort this out once and for all. Will reopen once I have complete fix ready.

Reply:
And [the unreadable Python installation folder issue](https://github.com/pyinstaller/pyinstaller/pull/4702#issue-380019672)? Where did that go?

Reply:
> And the unreadable Python installation folder issue? Where did that go?

That seems to kick in when you try to enter the folder using File Explorer. But it's possible to list the contents from terminal. Or from a python script. (Or perhaps things have changed over the course of last year?)

Reply:
Hmm, no clue. Given that my Windows partition hasn't been updated since 2012, I'm not the best source for reproducing this. Maybe it was more of a child-lock to stop people trying to uninstall software by deleting it and not a security thing after all. We can assume so until someone tells us otherwise.